collab-define/src/reminder.rs

```diff
@@ -174,6 +177,9 @@ fn reminder_from_map<T: ReadTxn>(txn: &T, map_ref: &MapRef) -> Result<Reminder>
   let is_ack = map_ref
     .get_bool_with_txn(txn, REMINDER_IS_ACK)
     .ok_or(anyhow::anyhow!("{} not found", REMINDER_IS_ACK))?;
+  let is_read = map_ref
```

Comment:
By default, the is_read is not set. You can return the default value, which is false.

Change
.ok_or(anyhow::anyhow!("{} not found", REMINDER_IS_READ))?; 
to:
.unwrap_or_default()

Reply:
Thanks